test/test.importance.jl

```diff
@@ -10,8 +10,8 @@
 
 using FeatureScreening: feature_importance
 using FeatureScreening.Types: FeatureSet, names
-using FeatureScreening: select, Top, Random, IndexBased, get_count
-using FeatureScreening: label, importance
+using FeatureScreening: TopSampler, RandomSampler, ImportanceSampler
```

Comment:
the order doesn't meet with our convention, you should reorder to alphabetical order.

Reply:
This entire project fails to follow our conventions in many respects, I just tried to blend in. Also, our conventions are in strong need of a rewrite, effort pending.